code/src/GridAPI/Sort.ts

```diff
@@ -8,41 +8,95 @@ namespace GridAPI.Sort {
      *
      * @export
      * @param {string} gridID ID of the Grid that is to be to check from results.
-     * @returns {*}  {void}
+     * @returns {*}  {string} Return Message containing the resulting code from sorting columns and the error message in case of failure
      */
-    export function Clear(gridID: string): void {
+    export function Clear(gridID: string): string {
         PerformanceAPI.SetMark('Sort.Clear');
 
-        if (!OSFramework.Helper.IsGridReady(gridID)) return;
-        const grid = GridManager.GetGridById(gridID);
+        let returnMessage = {
+            isSuccess: true,
+            message: 'Success',
+            code: OSFramework.Enum.ErrorCodes.GRID_SUCCESS
+        };
 
-        grid.features.sort.clear();
+        if (!OSFramework.Helper.IsGridReady(gridID)) {
+            returnMessage = {
+                isSuccess: false,
+                message: 'Grid not found',
+                code: OSFramework.Enum.ErrorCodes.CFG_GridNotFound
+            };
+            return JSON.stringify(returnMessage);
+        }
+        try {
+            GridManager.GetGridById(gridID).features.sort.clear();
+        } catch (error) {
+            returnMessage = {
+                isSuccess: false,
+                message: 'Error',
+                code: OSFramework.Enum.ErrorCodes.API_FailedClearSort
+            };
+        }
 
         PerformanceAPI.SetMark('Sort.Clear-end');
         PerformanceAPI.GetMeasure(
             '@datagrid-Sort.Clear',
             'Sort.Clear',
             'Sort.Clear-end'
         );
+
+        return JSON.stringify(returnMessage);
     }
 
+    /**
+     * Function that sorts a Grid column based in its ID and on a sorting
+     *
+     * @export
+     * @param {string} gridID ID of the Grid that is to be to check from results.
+     * @param {string} columnID
+     * @param {OSFramework.OSStructure.Sorting} sorting
+     * @return {*}  {string} Return Message containing the resulting code from sorting columns and the error message in case of failure
+     */
     export function ColumnSort(
         gridID: string,
         columnID: string,
         sorting: OSFramework.OSStructure.Sorting
-    ): void {
+    ): string {
         PerformanceAPI.SetMark('Sort.ColumnSort');
 
-        if (!OSFramework.Helper.IsGridReady(gridID)) return;
-        const grid = GridManager.GetGridById(gridID);
+        let returnMessage = {
+            isSuccess: true,
+            message: 'Success',
+            code: OSFramework.Enum.ErrorCodes.GRID_SUCCESS
+        };
 
-        grid.features.sort.sortColumn(columnID, sorting);
+        if (!OSFramework.Helper.IsGridReady(gridID)) {
+            returnMessage = {
+                isSuccess: false,
+                message: 'Grid not found',
+                code: OSFramework.Enum.ErrorCodes.CFG_GridNotFound
+            };
```

Comment:
here you can access to the properties of the type, e.g. `returnMessage.isSuccess = false`.
Also please change the name of the object to something different from "returnMessage" :)

Reply:
I'm following the same names that Gabriel is using for consistency. I believe we need to review all these changes on a separate task since there are more PRs following this approach.

Reply:
I'm not good with names..Any suggestion to replace "returnMessage" @rugoncalves ? :)

Reply:
This is done @rugoncalves! Only need a better name for the return object

Reply:
This is done @rugoncalves! Renamed to responseObj